geode-gfsh/src/distributedTest/java/org/apache/geode/management/internal/cli/NetstatDUnitTest.java

```diff
@@ -165,6 +166,7 @@ public void testOutputWithLsofToFile() throws Exception {
 
     File outputFile = new File(temp.newFolder(), "command.log.txt");
 
+    @SuppressWarnings("deprecation")
```

Comment:
Do we have a JIRA to replace this test with one using the non-deprecated code path? I understand that work is beyond the scope of 7869

Reply:
No. I think I am going to propose that any code changes after this point that produce a deprecation failure should be addressed correctly. Also many of these deprecated items will be stripped from the code in the next major, forcing this issue. The goal of this ticket is to not make this problem worse over time. A committer deprecating code will be forced to address it before it will build. If they use suppression it will be obvious in the PR and should be challenged.